parkhangs-expressapp/controllers/userController.js

```diff
@@ -89,7 +90,7 @@ const updateUser = async (req, res) => {
 
         if (body.savedParks !== undefined) {
             try {
-                await savePark(user.savedParks, body.savedParks)
+                await saveParks(user.savedParks, body.savedParks)
             } catch (error) {
                 console.error(error.message)
                 console.log("unable to update park favourite count")
```

Comment:
remove the 2 console statements

Reply:
these are so we can see the error in the console... necessary